contrib/IECoreUSD/src/IECoreUSD/PrimitiveAlgo.cpp

```diff
@@ -36,14 +36,23 @@
 
 #include "DataAlgo.h"
 
+#include "IECore/DataAlgo.h"
 #include "IECore/MessageHandler.h"
 
+#include "pxr/base/gf/matrix3f.h"
+#include "pxr/base/gf/matrix3d.h"
+#include "pxr/base/gf/matrix4f.h"
+#include "pxr/base/gf/matrix4d.h"
+
```

Comment:
Are these meant to be added here?

Reply:
Yes. Up to this point the matrix types have only been forward-declared, and we need the full definition to instantiate `VtValueFromExpandedData`.